src/store/validation/index.ts

```diff
@@ -42,31 +42,19 @@ export const stateFactory = (): IValidationState => ({
 })
 
 export const getters: GetterTree<IValidationState, IRootState> = {
-  /**
-   * Human readable errors, keys are index like `flow/flowId/path/to/prop`
-   * check this repo to see more available example: https://github.com/ajv-validator/ajv-i18n/blob/master/messages/index.js
-   * eg:
-   * {
-   *   "flow/1607666a-2216-4a17-b8ba-9eb861d72933/blocks/0/name": "should match pattern \"^[a-zA-Z_]\\w*$\"",
-   *   "flow/1607666a-2216-4a17-b8ba-9eb861d72933/languages/0/iso_639_3": "should match pattern \"^[a-z][a-z][a-z]$\"",
-   *   "block/a520eb17-49f0-4617-af18-b0d524f921ce#/required": "should have required property 'config', for params {\"missingProperty\":\"config\"}",
-   *   "block/a520eb17-49f0-4617-af18-b0d524f921ce/name": "should match pattern \"^[a-zA-Z_]\\w*$\"",
-   *   "block/a520eb17-49f0-4617-af18-b0d524f921ce/exits/1/tag": "should match pattern \"^[a-zA-Z_]\\w*$\"",
-   *   "block/44663ef4-5e1f-4ead-a8e5-05d6f7d47f0d/name": "should match pattern \"^[a-zA-Z_]\\w*$\""
-   * }
-   *
-   * Note that indexedErrors has more elements than validationStatuses.
-   */
-  flattenErrorMessages(state): IIndexedString {
-    const accumulator: IIndexedString = {}
+  validationStatusForMessageKey: (state) => (messageKey: string): ErrorObject | null => {
```

Comment:
This would work, but I'd rather avoid using param in a getter, in fact we cannot see the current value of the computed data as it is a function:
<img width="693" alt="Screen Shot 2021-08-24 at 7 58 33 PM" src="https://user-images.githubusercontent.com/68745918/130714060-1d2ddeae-a4bf-4289-8ca5-7d6040090ff8.png">

Or is there any solution to check current value in vue dev-tool ?

@bzabos , could you pls recall us your point on why you're not a big fan of `getters with params` too ? I just forgot what you said last year 😸

Reply:
You are right!  I went through the documentation which confirms that passing a parameter to a Vuex getter recomputes every time without cacheing.
I made changes to fetch an object and access it using the `messageKey` as you suggested

Reply:
Thanks for double checking from the documentation ;)